### PR TITLE
Hdm 472 date ingested and last modified

### PR DESCRIPTION
**Pull request by amolmkhedkar**

### CODE REVIEW

File: app/models/solr_document.rb [hunk 1]
```diff
@@ -43,6 +43,14 @@ def mdpi_timestamp
...
     fetch(Solrizer.solr_name(:original_checksum, :symbol), [])
   end
+  
+  def date_uploaded
+    fetch(Solrizer.solr_name(:system_create_dtsi, :stored_sortable), [])
+  end
+  
+  def date_modified
+    fetch(Solrizer.solr_name(:system_modified_dtsi, :stored_sortable), [])
+  end
```
Comment (afred) [inline app/models/solr_document.rb]: @amolmkhedkar If you use `Solr.solr_name`, then you don't need to include the dynamic solr suffix `_dtsi`. In fact, the method `Solr.solr_name` exists so you don't have to remember those suffixes, and so you don't have to change that code if the suffixes change.

**Comment by afred** [comment]
This will need a rebase as well.

File: app/models/solr_document.rb (hunk 1 above)
Comment (amolmkhedkar) [inline app/models/solr_document.rb]: Yes, I shall remove the suffix.

**Comment by amolmkhedkar** [comment]
I am trying to rebase using this on my local machine:

git fetch origin
git checkout -b HDM-472_date_ingested_and_last_modified origin/HDM-472_date_ingested_and_last_modified
git merge iu-demo

**Comment by afred** [comment]
@amolmkhedkar If you want to rebase, you'd use `git rebase iu-demo`, instead of `git merge iu-demo`.

Also.. after you fetch, you should be able to simply `git checkout HDM-472_date_ingested_and_last_modified`. You should not have to pass the `-b` option, nor should you have to specify the branch's upstream with `origin/HDM-472_date_ingested_and_last_modified`

**Pull request merged**